README.md

```diff
@@ -5,7 +5,7 @@
 This is an angular library that provides components to view and annotate PDF documents, as well as view images.
 
 ## Running demo app
-- npm run package
+- npm run package (deprecated)
```

Comment:
you can just replace that with npm install

Reply:
is not needed to run the local build but it is used in travis

Reply:
no, I just meant in the readMe itself, for someone trying to run the app